File: M1-fundamentos/B4-Introducao-JavaScript-e-Logica-de-Programacao/dia-4-javascript-objetos-e-funcoes/parte-2-funçoes/exercicio-4.js
function maiorCaracter(arr){
  let maior = '';
  for (let i = 0; i < arr.length; i++){
    if (arr[i].length >= maior.length){
      maior = arr[i];
    }
  }
  return maior;
}

console.log(maiorCaracter(['José', 'Lucas', 'Nádia', 'Fernanda', 'Cairo', 'Joana']));

File: M4-ciencia-da-computacao/b1-introducao-a-python/dia-01-aprendendo-python/exercicios/exercicio-4.py
def biggerName(names: list):
  countCharList = []
  for name in names:
    countCharList.append(len(name))

  position = countCharList.index(max(countCharList))

  return names[position]

names = ["José", "Lucas", "Nádia", "Fernanda", "Cairo", "Joana", "BabaGanushe"]
print(biggerName(names))